File: app/src/main/java/com/lkms/data/model/java/Role.java
package com.lkms.data.model.java;

import com.google.gson.annotations.SerializedName;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * Đại diện cho một hàng trong bảng "Role".
 */
@Data
@NoArgsConstructor
@AllArgsConstructor
public class Role {

    @SerializedName("roleId")
    private Integer roleId;

    @SerializedName("roleName")
    private String roleName;

    @SerializedName("roleStatus")
    private String roleStatus;
}

File: app/src/main/java/com/lkms/data/model/kotlin/Role.kt
package com.lkms.data.model.kotlin

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class Role(
    @SerialName("roleId")
    val roleId: Int? = null,

    @SerialName("roleName")
    val roleName: String? = null,

    @SerialName("roleStatus")
    val roleStatus: String? = null
)
